WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -3203,40 +3130,21 @@ function WeakAuras.ReloadTriggerOptions(data)
       WeakAuras.UpdateDisplayButton(data);
       WeakAuras.ReloadTriggerOptions(data);
     end
+    trigger_options = union(trigger_options, WeakAuras.GetGenericTriggerOptions(data, trigger, untrigger));
     local triggernum = optionTriggerChoices[id];
     if(trigger.type == "aura") then
       local aura_options = WeakAuras.GetBuffTriggerOptions(data, trigger);
-      displayOptions[id].args.trigger.args = union(trigger_options, aura_options);
+      trigger_options = union(trigger_options, aura_options);
     elseif(trigger.type == "event" or trigger.type == "status") then
       if(WeakAuras.event_prototypes[trigger.event]) then
-        if(trigger.event == "Combat Log") then
```

Comment:
Same reasoning as in the multi selection code path. ConstructOption does not care about subeventPrefix nor subEventSuffix